App.tsx

```diff
@@ -7,6 +7,7 @@ import { ThemeProvider } from "@contexts/theme";
 import { AuthProvider } from "@contexts/auth";
 
 import "expo-dev-client"; // Improve debugging when duing dev-client
+import "react-native-gesture-handler";
```

Comment:
This should be at the top, as the very first import: https://reactnavigation.org/docs/stack-navigator/#installation